deb/common/rules

```diff
@@ -110,6 +107,10 @@ override_dh_install:
 	dh_apparmor --profile-name=docker-ce -pdocker-ce
 
 override_dh_gencontrol:
+	# Use separate version for the buildx-plugin package, then generate the other control files as usual
+	# TODO override "Source" field in control as well (to point to buildx, as it doesn't match the package name)
```

Comment:
We should probably (in a follow-up) duplicate the current `deb/common` directory for each of the packages we build, and split the control file accordingly. That will be a bit of work though (also splitting make targets etc etc)

Ideally, have separate stages in the Dockerfile to collect the source-files (and dependencies) for each, also to get out of the current situation where we bind-mount things and build packages in a `docker run` (which is horrible, because not only do we lose (e.g.) modules cache, but also currently requires some hacks `chown` / `chmod` to be done to adjust file permissions for cleaning up 😞

Reply:
Yeah I already have a draft branch for that :eyes: